code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -143,6 +155,9 @@
 		if(ALIEN_BODYPART,LARVA_BODYPART) //aliens take double burn
 			burn *= 2
 
+	if(prob(brute*2))
```

Comment:
I'd like to see something a bit more comprehensive than just a percentage chance ideally. Only high-damage weapons should break bones in my opinion. Maybe something like `if(brute >= 15 && prob(brute*2))` stabbing someone with a scalpel should not break bones.